code/game/gamemodes/game_mode.dm

```diff
@@ -272,8 +272,9 @@ Implants;
 	return 0
 
 
-/datum/game_mode/proc/send_intercept()
-	var/intercepttext = "<FONT size = 3><B>Cent. Com. Update</B> Requested status information:</FONT><HR>"
+/datum/game_mode/proc/send_intercept() //Disabled.
+	return
+/*	var/intercepttext = "<FONT size = 3><B>Cent. Com. Update</B> Requested status information:</FONT><HR>"
 	intercepttext += "<B> In case you have misplaced your copy, attached is a list of personnel whom reliable sources&trade; suspect may be affiliated with the Syndicate:</B><br>"

```

Comment:
Also because it encouraged people to set their character's "Nanotrasen Relationship" to "opposed", as it would increase their chance of being on the report and getting free TCs.